lib/feature.pm

```diff
@@ -562,6 +562,57 @@ also does the equivalent of C<use strict>; see L<perlfunc/use> for details.
 
 =back
 
+=head1 CHECKING FEATURES
+
+C<feature> provides some simple APIs to check which features are enabled.
+
+=over
+
+=item feature::feature_enabled($feature)
```

Comment:
It produces nicer deeplinks if the =item is just the bare function name, and the way to call it is provided in a verbatim block below the heading.

Reply:
changed, I've added sample code, and added a note that callers will need to use feature (); if they don't otherwise load it.

thanks for the review.